std/process.d

```diff
@@ -3357,15 +3363,29 @@ static:
         // empty.
     }
     ---
+    Params:
+        name = name of the environment variable to retrieve
+        defaultValue = default value to return if the environment variable doesn't exist.
 
-    Throws:
-    $(REF UTFException, std,utf) if the variable contains invalid UTF-16
-    characters (Windows only).
+    Returns:
+        the value of the environment variable if found, otherwise
+        `null` if the environment doesn't exist.
     */
-    string get(in char[] name, string defaultValue = null) @safe
+    string get(in char[] name, string defaultValue = null) @safe nothrow
     {
         string value;
-        auto found = getImpl(name, value);
+        bool found;
+        version(Windows)
+        {
+            try
+                found = getImplWindows(name, value);
+            catch (WindowsException) {}
```

Comment:
Is this really enough to shut up the compiler? I mean, you're not handling all exceptions...

Reply:
answer: it's not :)
https://auto-tester.puremagic.com/show-run.ghtml?projectid=1&runid=3020178&isPull=true